client/components/chart/utils.js

```diff
@@ -134,8 +139,13 @@ export const getXLineScale = ( uniqueDates, width ) =>
  * @param {array} lineData - from `getLineData`
  * @returns {number} the maximum value in the timeseries multiplied by 4/3
  */
-export const getYMax = lineData =>
-	Math.round( 4 / 3 * d3Max( lineData, d => d3Max( d.values.map( date => date.value ) ) ) );
+export const getYMax = lineData => {
+	const yMax = Math.round(
+		4 / 3 * d3Max( lineData, d => d3Max( d.values.map( date => date.value ) ) )
+	);
+	const pow3Y = Math.pow( 10, ( ( Math.log( yMax ) * Math.LOG10E + 1 ) | 0 ) - 2 ) * 3;
```

Comment:
I think a comment here might help making this operation easier to understand.

Another solution would be splitting it into another function and adding a descriptive name to it:
```JS
const convertToPowerOf10 = value => {
	const pow3Y = Math.pow( 10, ( ( Math.log( value ) * Math.LOG10E + 1 ) | 0 ) - 2 ) * 3;
	return Math.ceil( value / pow3Y ) * pow3Y;
};

export const getYMax = lineData => {
	const yMax = 4 / 3 * d3Max( lineData, d => d3Max( d.values.map( date => date.value ) ) );
	return convertToPowerOf10( yMax );
}
```

(Probably a better name can be found)

What do you think? :slightly_smiling_face:

I also think the `Math.round()` around `yMax` value can be removed, now.

Reply:
Improved in https://github.com/woocommerce/wc-admin/pull/350/commits/27c85371eeac7f1252b2c5d5d6958deb9f5112cb